sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -663,3 +663,17 @@ def test_deprecated_base_estimator_parameters_can_be_set():
 
     with pytest.warns(FutureWarning, match="Parameter 'base_estimator' of"):
         clf.set_params(base_estimator__max_depth=2)
+
+
+# TODO(1.6): remove
+def test_deprecated_samme_r_algorithm():
+    adaboost_clf = AdaBoostClassifier(
+        estimator=DecisionTreeClassifier(max_leaf_nodes=4, random_state=1),
+        n_estimators=200,
+        random_state=42,
+    )
```

Comment:
Usually, we want unit tests to be as fast as possible. Currently, we only want to be sure that a warning is raised when `fit` is called without changing the default. So no need to wait for the 200 estimators to be fitted.

In the case that we check statistical property then we might not have any workaround and the tests are designed to be slower for a specific purpose.

```suggestion
    adaboost_clf = AdaBoostClassifier(n_estimators=1)
```

Reply:
Got it, thank you!